six/include/datadog_agent_six.h

```diff
@@ -46,7 +46,7 @@ DATADOG_AGENT_SIX_API void set_module_attr_string(six_t *, char *, char *, char
 // CONST API
 DATADOG_AGENT_SIX_API int is_initialized(six_t *);
 DATADOG_AGENT_SIX_API six_pyobject_t *get_none(const six_t *);
-DATADOG_AGENT_SIX_API const char *get_py_version(const six_t *);
+DATADOG_AGENT_SIX_API py_info_t *get_py_info(six_t *);
```

Comment:
same question about why the `const`ness was dropped, but on the `six_t*` parameter this time :)

Reply:
we cast six as a pointer to `two` or `three`, you can only call const method on a const six pointer. Since `get_py_info` is not const, so it `six_t`.